litellm/caching.py

```diff
@@ -930,22 +930,45 @@ def __init__(
             "index": {
                 "name": "litellm_semantic_cache_index",
                 "prefix": "litellm",
-                "storage_type": "hash",
+                # "key_separator": ":",
```

Comment:
Are these cahnges required ? These are in RedisSemanticCache class, isn't it ? We do not need to modify anything in RedisSemanticCache class.